snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedConstants.java

```diff
@@ -257,7 +257,6 @@ private Concepts() { }
 		public static final Set<String> HISTORICAL_ASSOCIATION_REFSETS = ImmutableSet.of(
 				REFSET_HISTORICAL_ASSOCIATION, 
 				REFSET_ALTERNATIVE_ASSOCIATION,
-				REFSET_MOVED_FROM_ASSOCIATION,
```

Comment:
I would like to see this set removed and new explicit sets introduced at the various places where we actually need historical association refsets. Refactor please.